_posts/2022-06-26-contextual-slu.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Incorporating context to improve SLU
+date: 2022-06-19
+tags: [slu, context, nlp]
+categories: [Machine Learning]
+# image: assets/images/label-noise-blog/label-noise.png
+layout: post
+authors: [sanchit-ahuja]
+latex: True
+---
+
+## Introduction
+In task-oriented dialogue systems, the spoken language understanding, or SLU, refers to the task of parsing the natural language utterances into
+semantic frames. The problem of contextual SLU majorly focuses on effectively incorporating dialogue information.
```

Comment:
are you focusing on just intent or also entities ?

Reply:
Only intent